calico_containers/pycalico/util.py

```diff
@@ -0,0 +1,12 @@
+def generate_cali_interface_name(prefix, ep_id):
```

Comment:
Is there no code somewhere already that does this that could be commonized?  How does felix usually decide which veth name to use?